pydicom/tests/test_valuerep.py

```diff
@@ -396,8 +396,7 @@ def test_valid(self, s: str):
             '1,000',              # no commas
             '1 000',              # no embedded spaces
             '127.0.0.1',          # not a number
-            '1.e',                # not a number
-            '',
```

Comment:
`is_valid_ds` is not used in pydicom. I changed it to see an empty string as valid, but I could also adapt it to behave as before - not sure what is better here. The description says "16 characters or fewer", so an empty string would match that...